Docs/General/ElectronicsCrashCourse.md

```diff
@@ -76,10 +80,21 @@ When you start up the robot you may see red error messages in the drive station
 If you see that, something might be broken on the CAN network.
```

Comment:
what is being communicated, what happens to the rest of the network if one goes down?